src/site/stages/build/drupal/graphql/nodeStepByStep.graphql.js

```diff
@@ -0,0 +1,54 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeStepByStep on NodeStepByStep {
+  ${entityElementsFromPages}
+  entityBundle
+
+  title
+  fieldIntroTextLimitedHtml {
+    processed
+  }
+  fieldAlertSingle {
+    entity {
+      ... alertSingle
+    }
+  }
+  fieldButtons {
+    entity {
+      ... button
+    }
+  }
+  fieldSteps {
+    entity {
+      ... on ParagraphStepByStep {
+        fieldStep {
+          entity {
+            ... on ParagraphStep {
+              fieldWysiwyg {
+                processed
+              }
+              fieldMedia {
+                entity {
+                  ... on Media {
+                    thumbnail {
+                      url
+                    }
+                  }
+                }
```

Comment:
I think you can do -

```suggestion
              fieldMedia {
                entity {
                  ... embeddedImage
                }
```

To make use of the reusable fragment. See `src/site/stages/build/drupal/graphql/page.graphql.js` for an example

Reply:
I got drupal errors trying to implement this:

![image](https://user-images.githubusercontent.com/12773166/92957347-9e98c200-f425-11ea-9e39-f259b26e2032.png)